example/org.eclipse.glsp.integration.workflow.editor/diagram/bundle.js

```diff
@@ -80869,7 +80962,7 @@ __webpack_require__(/*! core-js/es6/map */"./node_modules/core-js/es6/map.js");
 __webpack_require__(/*! core-js/es6/promise */"./node_modules/core-js/es6/promise.js");
 __webpack_require__(/*! core-js/es6/string */"./node_modules/core-js/es6/string.js");
 __webpack_require__(/*! core-js/es6/symbol */"./node_modules/core-js/es6/symbol.js");
-module.exports = __webpack_require__(/*! /home/tobias/Git/OpenSource/glsp-eclipse-integration/example/client/lib/main */"./lib/main.js");
+module.exports = __webpack_require__(/*! /home/johannes/Git/GLSP/glsp-eclipse-integration/example/client/lib/main */"./lib/main.js");
```

Comment:
We should probably gitignore this file. Otherwise we might run into lots of unnecessary/avoidable conflicts

Reply:
Will do